index.d.ts

```diff
@@ -39,7 +40,8 @@ interface Options {
   /**
    * @type {IncludeAndExclude}
    */
-  include?: string;
+	include?: string;
```

Comment:
this is a tab, but the other lines are spaces

Reply:
thanks